src/Peer.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1); // @codeCoverageIgnore
+
+namespace Rinq;
+
+use Rinq\Exception\InvalidNamespaceException;
+use Rinq\Ident\PeerId;
+
+/*
+ * Peer represents a connection to a Rinq network.
+ *
+ * Peers can act as a server, responding to application-defined commands.
+ * Use Peer::listen() to start accepting incoming command requests.
+ *
+ * Command requests are sent by sessions, represented by the Session interface.
+ * Sessions can also send notifications to other sessions. Sessions are created
+ * by calling Peer::session().
+ *
+ * Each peer is assigned a unique ID, which is represented by the PeerID class.
```

Comment:
We could annotate this here, unlike in Go  `represented by {@see PeerId}.` (note lowercase "d")